eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -243,5 +246,7 @@ page at http://checkstyle.sourceforge.net/config.html -->
         <!-- See http://checkstyle.sf.net/config_misc.html -->
         <module name="ArrayTypeStyle"/>
 
+        <!-- CUSTOM CHECKS -->
+        <module name="com.azure.customcheckstyles.ServiceClientSubclassCheck"/>
```

Comment:
Include a comment here about what this module does (basically, same as the class javadoc you wrote)